core/src/main/resources/lib/form/section.jelly

```diff
@@ -31,6 +31,9 @@ THE SOFTWARE.
       The section header text.
       If null is given, the entire &lt;f:section> tag becomes no-op.
     </st:attribute>
+    <st:attribute name="icon">
+      Optional attribute to set an icon for the section, this is only visible when using section-to-sidebar-items.js
```

Comment:
> this is only visible when using section-to-sidebar-items.js

#4733 / [JENKINS-62371](https://issues.jenkins.io/browse/JENKINS-62371)

Reply:
:), I wouldn't recommend adding to sections it would be too distracting.

This is somewhat accomplishing what the previous PR was for I think though.